test/integration_test/nearsync_application_test.go

```diff
@@ -0,0 +1,126 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+type configTestAppWithNearSync struct {
+	appKey               string
+	stressAppKey         string
+	instanceId           string
+	repl                 int
+	nearsyncReplStrategy string
+}
+
+func TestApplicationsWithNearSync(t *testing.T) {
+
+	// create secret on source and destination
+	funcCreateSecret := func() {
```

Comment:
nit: just call it `createSecret()`

Reply:
Because its called twice, once on source and once on destination